src/ClientResumeStatusCallback.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <folly/ExceptionWrapper.h>
+
+namespace reactivesocket {
+
+class ClientResumeStatusCallback {
+public:
+
+  // Called when a RESUME_OK frame is received during resuming operation
+  virtual void onResumeOk() = 0;
+
+  // Called when an ERROR frame with CONNECTION_ERROR is received during resuming operation
+  virtual void onConnectionError(folly::exception_wrapper ex) = 0;
+};
+
+class ClientResumeStatusCallbackDefault : public ClientResumeStatusCallback {
```

Comment:
just a nit, but are you used to naming things in the opposite order? Or why not just DefaultClientResumeStatusCallback
(To be consistent, I would also prefer word Handler instead of Callback, because we have RequestHandler and not RequestCallback. But will leave it up to you)